src/Microsoft.PowerShell.Security/security/CertificateCommands.cs

```diff
@@ -212,26 +212,20 @@ protected override void ProcessRecord()
 
         private static X509Certificate2 GetCertFromPfxFile(string path)
         {
-            X509Certificate2 cert = new X509Certificate2();
-
-            cert.Import(path);
-
+            X509Certificate2 cert = new X509Certificate2(path);
             return cert;
         }
 
         private static X509Certificate2 GetCertFromPfxFile(string path, SecureString password)
         {
-            X509Certificate2 cert = new X509Certificate2();
-
             //
             // NTRAID#DevDiv Bugs-33007-2004/7/08-kumarp
             // the following will not be required once X509Certificate2.Import()
             // accepts a SecureString
             //
-            string clearTextPassword = SecurityUtils.GetStringFromSecureString(password);
-
-            cert.Import(path, clearTextPassword, X509KeyStorageFlags.DefaultKeySet);
-
+            string clearTextPassword = Utils.GetStringFromSecureString(password);
+            
+            var cert = new X509Certificate2(path, clearTextPassword, X509KeyStorageFlags.DefaultKeySet);
```

Comment:
Thanks for noticing this. We need to update our code to use the new APIs once we move to a newer version of .NET Core packages. I opened this issue #3228 to track this effort as well as the list of similar code instances. Please feel free to update the list when you spot one.